flexmeasures/data/models/planning/battery.py

```diff
@@ -45,6 +47,12 @@ def schedule_battery(
     if roundtrip_efficiency <= 0 or roundtrip_efficiency > 1:
         raise ValueError("roundtrip_efficiency expected within the interval (0, 1]")
 
+    # Check for min and max SOC, or get default from sensor
+    if soc_min is None:
+        soc_min = sensor.get_attribute("min_soc_in_mwh")
```

Comment:
Why isn't `0` also the fallback value here, as in the charging station code? ("Can't drain the EV battery by more than it contains")

Reply:
Because of the block `# Check for required Sensor attributes`, which already guarantees that such a fallback value will not be used.